Matchismo/Model/PlayingCard.m

```diff
@@ -0,0 +1,53 @@
+//
+//  PlayingCard.m
+//  Matchismo
+//
+//  Created by Yeliena Khaletska on 28.05.2024.
+//
+
+#import "PlayingCard.h"
+
+@implementation PlayingCard
+
+- (NSString *)contents
+{
+    NSArray *rankStrings = [PlayingCard rankStrings];
+    return [rankStrings[self.rank] stringByAppendingString:self.suit];
```

Comment:
I always prefer to double-check that the index is not more than the array count. Here it's optional, but better to always keep this in mind 😇